fuzz/fuzz_targets/chanmon_fail_consistency.rs

```diff
@@ -688,10 +639,6 @@ pub fn do_test(data: &[u8]) {
 					chan_a_disconnected = true;
 					drain_msg_events_on_disconnect!(0);
 				}
-				let (new_node_a, new_monitor_a) = reload_node!(node_a_ser, 0, monitor_a);
```

Comment:
Wait, reload_node! does a thing, can't just completely remove it...

Reply:
oops! fixed